packages/react-api/src/Api.tsx

```diff
@@ -183,15 +183,27 @@ async function loadOnReady (api: ApiPromise, endpoint: LinkOption | null, inject
   };
 }
 
+function getWellKnownChain (chain = 'polkadot') {
+  switch (chain) {
+    case 'kusama':
+      return WellKnownChain.ksmcc3;
+  case 'polkadot':
+    return WellKnownChain.polkadot;
+  case 'rococo':
+    return WellKnownChain.rococo_v2_1;
+  case 'westend':
+    return WellKnownChain.westend2;
+  default:
+    throw new Error(`Unable to construct light chain ${chain}`);
```

Comment:
```suggestion
  switch (chain) {
    case 'kusama':
      return WellKnownChain.ksmcc3;
    case 'polkadot':
      return WellKnownChain.polkadot;
    case 'rococo':
      return WellKnownChain.rococo_v2_1;
    case 'westend':
      return WellKnownChain.westend2;
    default:
      throw new Error(`Unable to construct light chain ${chain}`);
  }
```

Sorry, I _did_ edit it, but it seems to have been lost, you were too quick :)